README.md

```diff
@@ -405,6 +486,30 @@ lager:trace_console([{request, '>', 117}, {request, '<', 120}])
 
 Using `=` is equivalent to the 2-tuple form.
 
+### Multiple sink support
+
+If using multiple sinks, there are limitations on tracing that you
+should be aware of.
+
+Traces are specific to a sink, which can be specified via trace
+filters:
+
+```erlang
+lager:trace_file("log/security.log", [{sink, audit}, {function, myfunction}], warning)
+```
+
+If no sink is thus specified, the default lager sink will be used.
+
+This has two ramifications:
+
+* Traces cannot intercept messages sent to a different sink.
+* Tracing to a file already opened via `lager:trace_file` will only be
+  successful if the same sink is specified.
+
+The former can be ameliorated by opening multiple traces; the latter
+can be fixed by rearchitecting lager's file backend, but this has not
+been tackled.
+
```

Comment:
Curious if we could support {sink, [audit, lager, foo, bar]} in a future release, to allow multiple sinks writing to a single trace file? Or would this require too much synchronization overhead to prevent overlapping writes on the trace file?